import-export-cli/cmd/addEnv.go

```diff
@@ -41,12 +42,16 @@ const addEnvCmdLongDesc = "Add new environment and its related endpoints to the
 const addEnvCmdExamples = utils.ProjectName + ` ` + AddCmdLiteral + ` ` + AddEnvCmdLiteralTrimmed + ` production \
 --apim  https://localhost:9443 
 
+` + utils.ProjectName + ` ` + AddCmdLiteral + ` ` + AddEnvCmdLiteralTrimmed + ` dev \
```

Comment:
Shall we add an example with the minimum arguments need to provide when adding a new environment with both APIM and MI instances?

```
 utils.ProjectName + ` ` + AddCmdLiteral + ` ` + AddEnvCmdLiteralTrimmed + ` prod \
--apim  https://apim.com:9443 \
--mi https://localhost:9164
```

Reply:
Fixed with 224f0cc